hack/setup_env.sh

```diff
@@ -40,11 +40,13 @@ function spectral() {
 }
 
 function assisted_service() {
+  mkdir $GOPATH
   latest_kubectl_version=$(curl --retry 5 -L -s https://dl.k8s.io/release/stable.txt)
   curl --retry 5 -L "https://dl.k8s.io/release/${latest_kubectl_version}/bin/linux/amd64/kubectl" -o /tmp/kubectl && \
     install -o root -g root -m 0755 /tmp/kubectl /usr/local/bin/kubectl && \
     rm -f /tmp/kubectl
-  yum install -y docker podman libvirt-clients awscli python3-pip postgresql genisoimage skopeo p7zip && \
+  
+  dnf -y update && dnf install -y golang which jq docker python3 podman-remote libvirt-client awscli python3-pip postgresql genisoimage skopeo p7zip && \
```

Comment:
Isn't `docker` conflicting with the `podman` image here?

Reply:
no, it's tested